### DISCUSSION TITLE
[Feature Request] Taskbar Background Helper; add backgorund while windows virtual desktop is opened

### DISCUSSION THREAD

**Discussion by sapphired18** [ramensoftware/windhawk-mods#1166] [Mod ideas]
Add option to trigger taskbar background color while in Windows Virtual Desktop

> https://github.com/user-attachments/assets/ba547a41-b0e4-458e-b7eb-b9ad8c8191f2

**Comment by m417z** [comment]
Done in version 1.0.2.
